package.json

```diff
@@ -66,7 +66,7 @@
     "json-schema": "^0.3.0",
     "npm-check-updates": "^11",
     "prettier": "^2.3.1",
-    "projen": "^0.24.6",
+    "projen": "^0.22.15",
```

Comment:
I suppose this rollback was not intentional...

Reply:
make sure to delete `yarn.lock` and `node_modules` then remove changes from package.json, then run `yarn install --frozen-lockfile` before doing `yarn projen`. If you have an old version of projen locally that you're running against it will downgrade.